src/bdc/steps/google_places.py

```diff
@@ -18,13 +18,14 @@
 
 
 class GooglePlaces(Step):
-    # TODO: replace the requests package with the python client for Google Maps Service https://github.com/googlemaps/google-maps-services-python
     name = "Google_Places"
     URL = "https://maps.googleapis.com/maps/api/place/textsearch/json?query="
     fields = ["business_status", "formatted_address", "name", "user_ratings_total"]
+    gmaps = None
 
     def load_data(self) -> None:
-        pass
+        # don't perform this in class body or else it will fail in tests due to missing API key
+        self.gmaps = googlemaps.Client(key=GOOGLE_PLACES_API_KEY)
```

Comment:
Does this throw an exception when the Key is invalid or not set? Maybe this is the place to throw a StepError?

Reply:
It does. This should only ever happen in the test env but you never know. I'll check if an API key is present in verify() and throw an error in `load_data()` if it's somehow still missing.

Reply:
Okay is this also robust against the key being wrong?
It should not happen really but maybe it would be more error resilient

Reply:
Yes with a wrong key, all the entries would result in being None basically.

Reply:
okay then I approve the PR as well :)